fdbclient/ServerKnobs.cpp

```diff
@@ -851,12 +851,13 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( LATENCY_METRICS_LOGGING_INTERVAL,                     60.0 );
 
 	// Cluster recovery
-	init ( CLUSTER_RECOVERY_EVENT_NAME_PREFIX,               "Master");
+	init ( CLUSTER_RECOVERY_EVENT_NAME_PREFIX,              "Master" );
 
     // encrypt key proxy
-    init( ENABLE_ENCRYPTION,                                   false );
-    init( ENCRYPTION_MODE,                              "AES-256-CTR");
-    init( SIM_KMS_MAX_KEYS,                                      4096);
+    init( ENABLE_ENCRYPTION,                                   false ); if ( randomize && BUGGIFY ) { ENABLE_ENCRYPTION = deterministicRandom()->random01(); }
```

Comment:
Is following scenario possible: cluster booted up with _enable_encryption_ but then got bounced and restarted with _no_encryption_enable_; if yes, how would it handle to recover the encrypted data?

Reply:
I think in this case it is expected that recover will fail with `encrypt_key_not_found`, since EKP is not available. Do you have any suggestion to handle this case?